cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -100,6 +117,67 @@ def optimization_at(
         )
 
 
+class StrategyExecutorTransformer:
+    """Executes an acquaintance strategy."""
+
+    def __init__(self, execution_strategy: ExecutionStrategy) -> None:
+        """Initializes transformer.
+
+        Raises:
+            ValueError: if execution_strategy is None.
+        """
```

Comment:
nit: Add `Args` section to init docstring. 
```suggestion
        """Initializes transformer.
        
        Args:
            execution_strategy: The `ExecutionStrategy` to execute. 

        Raises:
            ValueError: if execution_strategy is None.
        """
```